core/src/components/item/item.tsx

```diff
@@ -350,6 +350,22 @@ export class Item implements ComponentInterface, AnchorInterface, ButtonInterfac
     }
   }
 
+  private getFirstInteractive() {
+    if (Build.isTesting) {
+      /**
+       * Pseudo selectors can't be tested in unit tests.
+       * It will cause an error when running the tests.
+       *
+       * We can remove this once https://github.com/ionic-team/stencil/issues/3588 is fixed.
+       */
+      return undefined;
+    }
+    const controls = this.el.querySelectorAll<HTMLElement>(
+      'ion-toggle:not([disabled]), ion-checkbox:not([disabled]), ion-radio:not([disabled])'
```

Comment:
Should we also do this for `ion-select`? I'm observing this behavior there too, using the `select/item` test.

Reply:
Good catch! Yes, I think we should. I'll update the design doc & then make the changes here.